server/src/main/java/org/opensearch/index/ReplicationStats.java

```diff
@@ -76,9 +76,15 @@ public void writeTo(StreamOutput out) throws IOException {
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(Fields.SEGMENT_REPLICATION);
-        builder.field(Fields.MAX_BYTES_BEHIND, new ByteSizeValue(maxBytesBehind).toString());
-        builder.field(Fields.TOTAL_BYTES_BEHIND, new ByteSizeValue(totalBytesBehind).toString());
-        builder.field(Fields.MAX_REPLICATION_LAG, new TimeValue(maxReplicationLag));
+        if (builder.humanReadable()) {
```

Comment:
you can pass the fields in directly to builder.humanReadableField - [example](https://github.com/opensearch-project/OpenSearch/blob/d426801ae6753e25b8d9f45da36ca4652bad20d3/server/src/main/java/org/opensearch/index/engine/SegmentsStats.java#L248)

Reply:
On testing it out, doesn't seem to be a way to use this without breaking bwc.

Reply:
Lets do something like this -
https://github.com/opensearch-project/OpenSearch/blob/3899d117722e5517c3c02709cffc981bdf677fd5/server/src/main/java/org/opensearch/index/remote/RemoteSegmentStats.java#L296-L297

`builder.humanReadableField(<>,<>,<>)` - first 2 args are basically name of fields for raw and human readable fields. We can use the existing names for human readable fields and give rational names for raw fields.

Reply:
@mch2  lets add new fields and deprecate this in future releases -
```
builder.humanReadableField("max_lag", "max_lag_in_bytes", new ByteSizeValue(maxBytesBehind).toString());
builder.humanReadableField("total_lag", "total_lag_in_bytes", new ByteSizeValue(totalBytesBehind).toString());
builder.humanReadableField("max_lag_time", "max_lag_time_in_millis", new TimeValue(maxReplicationLag));
```
The above is over and above the existing fields. Constants to be replaced with static strings.

Reply:
This makes sense, lets correct it and deprecate.  Also nit I think the raw key comes first in .humanReadableField

Reply:
Thanks for the suggestion @ashking94! Made the changes, take another look.